### DISCUSSION TITLE
3.0版本无法切换2.0版本。访问不到 是服务挂了么

### DISCUSSION THREAD

**Discussion by cjzgitHub** [NervJS/taro#14980] [Q & A] [answered]
### 这个特性解决了什么问题？
 直接使用yarn global upgrade @tarojs/cli@2.2.17 无法切换 包括npm跟taro也试过了 是服务挂了么，请尽快解决

### 这个 API 长什么样？
请尽快解决

**Comment by ZakaryCode** [comment]
请检查网络

**Comment by cjzgitHub** [comment, marked as answer]
检查过了 是mac电脑不可以 但是window系统可以

**Reply by ZakaryCode** [reply]
与系统无关联，检查下网络链接，不行就切换镜像，网上搜一下应该有挺多教程的

**Reply by cjzgitHub** [reply]
我们2台mac同时都出现这个问题了 昨天还是好的哈
window的电脑就不会有这个问题，所以不是镜像的问题
